File: engine/src/main/java/client/database/data/AccountLoginData.java
package client.database.data;

import java.sql.Date;
import java.sql.Timestamp;

public class AccountLoginData {
   private int loggedIn;

   private Timestamp lastLogin;

   private Date birthday;

   public AccountLoginData(int loggedIn, Timestamp lastLogin, Date birthday) {
      this.loggedIn = loggedIn;
      this.lastLogin = lastLogin;
      this.birthday = birthday;
   }

   public int getLoggedIn() {
      return loggedIn;
   }

   public Timestamp getLastLogin() {
      return lastLogin;
   }

   public Date getBirthday() {
      return birthday;
   }
}

File: engine/model/src/main/scala/client/database/data/AccountLoginData.scala
package client.database.data

import java.sql.{Date, Timestamp}

class AccountLoginData(private var _loggedIn: Int, private var _lastLogin: Timestamp, private var _birthday: Date) {
  def loggedIn: Int = _loggedIn

  def lastLogin: Timestamp = _lastLogin

  def birthday: Date = _birthday
}
